src/dicts.jl

```diff
@@ -117,6 +117,58 @@ function Base.get(d::LMDBDict{K,V}, key, default) where {K,V}
     end
 end
 
+function Base.get!(d::LMDBDict{K,V}, key, default) where {K,V}
+    txn_dbi_do(d, readonly = true) do txn, dbi
+        mdb_key_ref = Ref(MDBValue(toref(convert(K,key))))
+        mdb_val_ref = Ref(MDBValue())
+        # Get value
+        ret = _mdb_get(txn.handle, dbi.handle, mdb_key_ref, mdb_val_ref)
+        if ret == MDB_NOTFOUND
+            d[key] = default
+            return default
+        elseif ret == Cint(0)
+            return convert(V, mdb_val_ref)
+        else
+            throw(LMDB.LMDBError(ret))
+        end
+    end
+end
+
+function Base.get(f::F, d::LMDBDict{K,V}, key) where {K,V,F<:Function}
```

Comment:
In the respective Base methods, usually `Base.get(f::Union{Function,Type},...` is used as the signature. I think it would be good to be consistent here to allow calling this method on callable structs as well.

Reply:
This is done now.